README.md

```diff
@@ -8,20 +8,52 @@
 
 The [official docs for the LogDNA terraform provider](https://registry.terraform.io/providers/logdna/logdna/latest/docs) can be found in the Terraform registry.
 
-## Example Terraform Configuration
+Using the `logdna_alert` resource, a user can create a Preset Alert with a provided `name`, `email_channel`, `pagerduty_channel` and `webhook_channel`, delete a Preset Alert with a given `presetid` or update a Preset Alert using the `presetid` and `name`.
+
```

Comment:
Please switch the order of the paragraphs about View and Presets, so that we start with talking about Views... then secondly about Preset.